contribs.md

```diff
@@ -38,13 +38,13 @@ In this command, we're syncing with the `main` branch. You can choose another on
 
 ### Contributing documentation
 
-All documents are written in Markdown, using a basic structure and stored as .md files. These are then pulled to [our.umbraco.com/documentation](https://our.umbraco.com/documentation) for browsing.
+All documents are written in Markdown, using a basic structure and stored as .md files. These are then pulled to [docs.umbraco.com/umbraco-cms](https://docs.umbraco.com/umbraco-cms/) for browsing.
```

Comment:
same suggestion here, it should link to main docs page docs.umbraco.com instead of CMS docs page, can you please make sure all the other changes links from our.umbraco.com/documentation to https://docs.umbraco.com? 😊

Reply:
This should be updated in the three files in commit https://github.com/umbraco/UmbracoDocs/pull/5341/commits/a267ca7ba5654b1ceab850d17b8c526ef442b6d6

Reply:
This should be updated in the commit https://github.com/umbraco/UmbracoDocs/pull/5341/commits/97cc85c3c7975ef8e6e7aa861d9d2e56a9f92f94